data/markdown_doc/default_template.erb

```diff
@@ -16,19 +16,101 @@
 
 ## Module Ranking
 
-<%= normalize_rank(items[:mod_rank]) %>
+<%# https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking %>
+**<%= items[:mod_rank_name] %>**
 
-## Side Effects
+<% if items[:mod_rank_name] == "Excellent" %>
+> The exploit will never crash the service. This is the case for SQL Injection, CMD execution, RFI, LFI, etc. No typical
+> memory corruption exploits should be given this ranking unless there are extraordinary circumstances.
 
-<%= normalize_side_effects(items[:mod_side_effects]) %>
+<% elsif items[:mod_rank_name] == "Great" %>
+> The exploit has a default target AND either auto-detects the appropriate target or uses an application-specific return
+> address AFTER a version check.
 
-## Reliability
+<% elsif items[:mod_rank_name] == "Good" %>
+> The exploit has a default target and it is the "common case" for this type of software (English, Windows 7 for a
+> desktop app, 2012 for server, etc).
 
-<%= normalize_reliability(items[:mod_reliability]) %>
+<% elsif items[:mod_rank_name] == "Normal" %>
+> The exploit is otherwise reliable, but depends on a specific version and can't (or doesn't) reliably autodetect.
 
-## Stability
+<% elsif items[:mod_rank_name] == "Average" %>
+> The exploit is generally unreliable or difficult to exploit.
 
-<%= normalize_stability(items[:mod_stability]) %>
+<% elsif items[:mod_rank_name] == "Low" %>
+> The exploit is nearly impossible to exploit (or under 50% success rate) for common platforms.
+
+<% elsif items[:mod_rank_name] == "Manual" %>
+> The exploit is unstable or difficult to exploit and is basically a DoS. This ranking is also used when the module has
+> no use unless specifically configured by the user (e.g.: [exploit/windows/smb/psexec][1]).
+
+<% end %>
+
+## Module Traits
+
```

Comment:
add link to https://github.com/rapid7/metasploit-framework/wiki/Definition-of-Module-Reliability,-Side-Effects,-and-Stability for consistency with the one to exploit rank